Client/WarFare/UIItemUpgrade.h

```diff
@@ -0,0 +1,139 @@
+﻿// CUIItemUpgrade.h: interface for the CUIItemUpgrade class.
+//
+//////////////////////////////////////////////////////////////////////
+
+#if !defined(AFX_UIItemUpgrade_H__fd9f5093_0ed3_4c08_9e31_19c40773b24d__INCLUDED_)
+#define AFX_UITRANSACTIONDLG_H__fd9f5093_0ed3_4c08_9e31_19c40773b24d__INCLUDED_
+
+#if _MSC_VER > 1000
+#pragma once
+#endif // _MSC_VER > 1000
+
+#include <N3Base/N3UIBase.h>
```

Comment:
There's no need to include control classes in the header.
Include them in the source file, so they don't pollute everything referencing this.

They can be (and are) forward-declared already.

N3UIWndBase.h also takes care of the <N3Base/N3UIBase.h> include, though using CN3UIWndBase here has questionable method as well.